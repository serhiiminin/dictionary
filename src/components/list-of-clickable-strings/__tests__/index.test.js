import React from 'react';
import renderer from 'react-test-renderer';
import ListOfClickableStrings from '..';


describe('List of clickable strings', () => {
  const component = renderer.create(
    <ListOfClickableStrings
      items={['One', 'two']}
      onClick={() => {}}
    />
  );
  let tree = component.toJSON();

  test('render', () => {
    expect(tree).toMatchSnapshot();
  });

  test('delimiter', () => {
    tree.props.delimiter=',';
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
