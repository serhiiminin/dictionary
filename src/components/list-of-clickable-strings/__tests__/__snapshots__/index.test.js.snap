// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List of clickable strings delimiter 1`] = `
<div>
  <a
    className="ListOfClickableStrings-clickableWord-0-1-1"
    href="/"
    onClick={[Function]}
  >
    One
  </a>
  <a
    className="ListOfClickableStrings-clickableWord-0-1-1"
    href="/"
    onClick={[Function]}
  >
    two
  </a>
</div>
`;

exports[`List of clickable strings render 1`] = `
<div>
  <a
    className="ListOfClickableStrings-clickableWord-0-1-1"
    href="/"
    onClick={[Function]}
  >
    One
  </a>
  <a
    className="ListOfClickableStrings-clickableWord-0-1-1"
    href="/"
    onClick={[Function]}
  >
    two
  </a>
</div>
`;
